help/en/html/hardware/cmri/CMRI.shtml

```diff
@@ -89,7 +91,10 @@ for easy access click an item in this list:</p>
       <li><a href="#L568">CMRInet Manager</a></li>
       <li><a href="#L580">CMRInet Packet Monitor</a></li>
       <li><a href="#L597">CMRInet Network Metrics</a></li>
+    </ul>
+    <ul>
```

Comment:
Why are you closing the list and immediately reopening it?

Reply:
Corrected lines in error.